e2e/terraform/packer/README.md

```diff
@@ -18,4 +18,47 @@ $ packer --version
 
 # build linux AMI
 $ packer build packer.json
+
+# build Windows AMI
+$ packer build packer-windows.json
+```
+
+## Debugging Packer Builds
+
+You'll need the Windows administrator password in order to access Windows machines via `winrm` as Packer does. You can get this by enabling `-debug` on your Packer build.
+
+```sh
+packer build -debug -on-error=abort packer-windows.json
+...
+==> amazon-ebs: Pausing after run of step 'StepRunSourceInstance'. Press enter to continue.
+==> amazon-ebs: Waiting for auto-generated password for instance...
+    amazon-ebs: Password (since debug is enabled): <redacted>
+```
+
+Alternately, you can follow the steps in the [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/retrieve-windows-admin-password/). Note that you'll need the `ec2_amazon-ebs.pem` file that Packer drops in this directory.
+
+
+Then in powershell (note the leading `$` here indicate variable declarations, not shell prompts!):
```

Comment:
Could also include basic vnc instructions here for folks less familiar with PS?

Reply:
Sure thing! As soon as I figure it out myself! 😀